Automattic/jetpack
# VideoPress: Add `poster` endpoint for updating and getting poster

This one includes a new endpoint (`/:guid/poster`) that will accept `GET` and `POST` calls to update and retrieve poster image.

Related with #26153 

#### Changes proposed in this Pull Request:

* Add new route
* Include utilities for tokens
* Avoid override default `Authorization` for `Connection` method

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Open `/wp-json/wp/v2/media` in your browser. 
* Choose one video and get it `videopress_guid`.
* Open/Download [Postman](https://www.postman.com/)
* Create a new POST Request for `/wp-json/wpcom/v2/videopress/:videopress_guid/poster`
* Select `raw` and `JSON` as body.
* Set the body for: `{ "at_time": 1000, "is_millisec": true }`
* Run the request.
* It should return `code:200` and `data` with `generating: true` - ⚠️ do not open the returned poster url, for some reason it breaks the generation.
* Change the request type for `GET`.
* Run it until `generating` becomes `false`.
* Now, open the poster, it should be a new image from `1sec` frame.
